tests/test_admin.py

```diff
@@ -2517,7 +2518,7 @@ def test_extended_version_change_list_display_renders_without_list_display(self)
         # Check list_action links are rendered
         self.assertContains(response, "cms-versioning-action-btn")
         self.assertContains(response, "cms-versioning-action-preview")
-        self.assertContains(response, "cms-versioning-action-edit")
+        self.assertContains(response, 'title="Edit"')
```

Comment:
This is a change I brought in from navigation, without fully thinking through, that also applies to the comment above regarding the edit param being sent to the context. I could make this consistent, although it will require updates to the relevant tests in versioning. I initially went for this approach, although it felt wrong to be making changes to a package to make it consistent with another, which I accidentally brought changes across from.

Reply:
It is added by this: https://github.com/django-cms/djangocms-versioning/blob/master/djangocms_versioning/templates/djangocms_versioning/admin/icons/base.html#L9

We are going to need the features from the base icon template because the external packages are going to want to control the icons in different ways, by adding custom classes this gives additional controls.